graphql2/graphqlapp/calendarsubscription.go

```diff
@@ -0,0 +1,98 @@
+package graphqlapp
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"github.com/target/goalert/calendarsubscription"
+	"github.com/target/goalert/graphql2"
+	"github.com/target/goalert/schedule"
+)
+
+type CalendarSubscription App
+
+func (a *App) CalendarSubscription() graphql2.CalendarSubscriptionResolver {
+	return (*CalendarSubscription)(a)
+}
+
+func (a *CalendarSubscription) ReminderMinutes(ctx context.Context, obj *calendarsubscription.CalendarSubscription) ([]int, error) {
+	var config calendarsubscription.Config
+	err := json.Unmarshal(obj.Config, &config)
+	obj.ReminderMinutes = config.ReminderMinutes
+	return obj.ReminderMinutes, err
+}
+func (a *CalendarSubscription) ScheduleID(ctx context.Context, obj *calendarsubscription.CalendarSubscription) (string, error) {
```

Comment:
Is this required? it should map to the `ScheduleID` field automatically (json definitions may be changing that)

Reply:
No not required, removed now. Was generated previously before ScheduleID was added to the CalendarSubscription struct.